queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01'
+
```

Comment:
- [ ] Please make sure to select all animals that have a birth date between the years `2016` and `2019`, meaning that if they had a birth date in the year 2019 (until `2019/31/12`) they should be selected.

Reply:
thanks for you help i will make sure to fix that issue